app/workers/hca/submission_failure_email_analytics_job.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+module HCA
+  class SubmissionFailureEmailAnalyticsJob
+    include Sidekiq::Worker
+
+    sidekiq_options(unique_for: 30.minutes, retry: false)
+
+    def initialize
+      Sentry::TagRainbows.tag
+
+      unless FeatureFlipper.send_email?
+        raise Common::Exceptions::ParameterMissing.new(
+          'GovDelivery token or server',
+          detail: 'It should be configured in settings.yml'
+        )
+      end
+      if Settings.google_analytics_tracking_id.blank?
+        raise Common::Exceptions::ParameterMissing.new(
+          'Google Analytics tracking ID',
+          detail: 'It should be configured in settings.yml'
+        )
+      end
+      @tracker = Staccato.tracker(Settings.google_analytics_tracking_id)
+      @time_range_start = 1445.minutes.ago
+      @time_range_end = 5.minutes.ago
+    end
+
+    def perform
+      page = 0
+      loop do
+        page += 1
+        hca_emails(page).each do |email|
+          eval_email(email)
+        end
+        break if we_should_break?
+      end
+    end
+
+    private
+
+    def we_should_break?
+      Time.zone.parse(@all_emails.collection.last.created_at) < @time_range_start || @all_emails.collection.count < 50
+    end
+
+    def hca_emails(page)
+      @all_emails = govdelivery_client.email_messages.get(
+        page: page,
+        sort_by: 'created_at',
+        sort_order: 'DESC',
+        page_size: 50
+      )
+      @all_emails.collection.select do |email|
+        [HCASubmissionFailureMailer::SUBJECT].include?(email.subject) &&
```

Comment:
Reference subject here now, and construct it this way (`Array#include?`) so that it's obvious how to include a second subject line if necessary.